website/sidebars.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     'interactors/quick-start',
     'interactors/built-in-dom',
     'interactors/locators-filters-actions',
+    'interactors/assertions-matchers',
```

Comment:
The sidebar ends up being not too useful imo. It seems just two groups of concepts but tell nothing to the reader that don't know them. Can we name each page after the problem they can learn to solve in each page instead of the functions that we'll cover?  

<img width="313" alt="Screenshot 2021-03-16 at 19 41 49" src="https://user-images.githubusercontent.com/4451393/111362932-ae70b780-868f-11eb-91b6-ba212facd9d7.png">

For instance, I'd go as far as saying "Asserting with Interactors" as the title for the second page. I'm not sure what'd be an accurate title for the first one.

Reply:
I think saying `Asserting with Interactors` will be redundant because this is the `Interactors` docs.

Reply:
@jorgelainfiesta what do you think about a sidebar like this? It would require minimal refactoring of the content itself. We want to include the terminology in the title, so that someone can say "That's a Matcher. There is a docs pages about them..." when someone is learning. But also, we can give a clue what they are for.

```
Quick start
Built in dom
Queries with Locators and Filters
User Actions
Assertions
Flexibility with Matchers
Write your own Interactor
Integrations
```

Reply:
Created issue [here[(https://github.com/thefrontside/bigtest/issues/907)